perfrunbook/debug_system_perf.md

```diff
@@ -125,3 +125,59 @@ When running Java applications, monitor for differences in behavior using JFR (J
   # Capture output from x86 and Graviton2 and then diff the files
   ```
 
+## Check Memory behavior via `sar -r`
+
+If your benchmark involves reading and writing to file, for example database workloads, check for "dirty pages" and "page cache" via `kbdirty`, `kbcached` respectively between two systems. This gives important information if your system is getting limited by page cache or not compared to other.
+
+**Dirty Pages**: Is the data stored in page cached waiting to be written to the underlying storage device.
+
+**kbdirty**: Is the amount of memory in kilobytes waiting to get written back to the disk.
+
+Keeping around less data in the "dirty" page cache means the system is writing to disk more often, which blocks other disk accesses such as reads from completing as quickly.
+To compare "dirty" pages related settings between systems, we can use the below command:
+```
+sysctl -a | grep dirty
+vm.dirty_background_ratio = 10
+vm.dirty_background_bytes = 0
+vm.dirty_ratio = 20
+vm.dirty_bytes = 0
+vm.dirty_writeback_centisecs = 500
+vm.dirty_expire_centisecs = 3000
+```
+`vm.dirty_background_ratio` is the percentage of system memory which when dirty, causes the system to start writing data to the disk.
+
+`vm.dirty_ratio` is the percentage of system memory which when dirty, causes the process doing writes to block and write out dirty pages to the disk.
+
+Depending on requirement, we may want to retest workload by increasing or decreasing dirty pages(kbdirty), in cases of large databases, it's recommended to keep these values low, to avoid I/O bottlenecks when the system load increases.
+We either tune`vm.dirty_bytes` and `vm.dirty_background_bytes` or `vm.dirty_ratio` and `vm.dirty_background_ratio`.
+If you set the `_bytes` version, the `_ratio` version will become 0, and vice-versa.
+
+`dirty_writeback_centisecs` can also be tuned to decide when to periodically wake up and write old data out to disk. Setting this to zero disables periodic writeback altogether.
+
+
+
+## Check performance between kernel versions
```

Comment:
In the configuring_your_sut.md file we should add steps on how to check for differing kernel configs, but not here.